packages/language-server/src/lib/MessageHandler.ts

```diff
@@ -213,7 +182,7 @@ export function handleHoverRequest(document: TextDocument, params: HoverParams):
     return
   }
 
-  const block = getModelOrTypeOrEnumOrViewBlock(word, lines)
+  const block = getDataBlock(word, lines)
```

Comment:
What does `data` means here?

Reply:
Maybe rename to `getDatamodelBlock`

Reply:
> What does `Data` means here?

It was supposed to be an analog to `configBlock`, but `Datamodel` also works